tools/test.py

```diff
@@ -255,6 +255,9 @@ def HasRun(self, output):
         logger.info('#' + l)
       for l in output.output.stdout.splitlines():
         logger.info('#' + l)
+    elif output.HasSkipped():
+      logger.info('ok %i - %s # skip %s' % (self._done, command,
+        output.output.stdout.replace('1..0 # Skipped:', '').strip()))
```

Comment:
@thefourtheye see link in PR up top.

I'll have a look at how other consumers parses skipped.